src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -159,7 +180,7 @@ type DefaultKeyRepository(seed: uint256) =
         member this.GenerateKeyFromBasePointAndSign(psbt, pubkey, basePoint) =
             use ctx = CryptoUtils.impl.newSecp256k1()
             let basepointSecret: Key = this.BasepointToSecretMap.TryGet pubkey
-            let priv2 = Generators.derivePrivKey ctx (basepointSecret)  basePoint 
+            let priv2 = Generators.derivePrivKey ctx (basepointSecret)  basePoint
```

Comment:
nit: no change in the above line ^

Reply:
A trailing space has been removed. Is that worth committing?

Reply:
I normally don't like to break git-blame because of this, but it is just me